packages/core/.storybook/preview-head.html

```diff
@@ -17,6 +17,7 @@
   body {
     font-family: "Figtree", sans-serif;
     box-sizing: border-box;
+    color: var(--sb-primary-text-color);
```

Comment:
Not sure why we use the --sb prefix, we can use our tokens, but it's out of scope